app/views/ht_users/edit.html.erb

```diff
@@ -64,21 +62,24 @@
         </dd>
         <dt>Renewal Status</dt>
         <dd><%= @user.badge %></dd>
-        <dt><%= form.label :iprestrict %></dt>
+        <dt><%= form.label :iprestrict, for: :iprestrict_field %></dt>
         <dd><%= form.text_field :iprestrict, value: @user.iprestrict&.join(', '), size: 40, id: :iprestrict_field, disabled: @user.mfa %>
           <p class="text-muted"><%= t('ht_user.edit.iprestrict_prompt') %></p>
         </dd>
-        <dt>Multi-Factor?:</dt>
         <% if @user.ht_institution.shib_authncontext_class.present? %>
+          <dt><%= form.label 'Multi-Factor?:', for: :mfa_checkbox %></dt>
```

Comment:
Also worth pulling out this logic to a presenter?

Reply:
While I'm at it...